### DISCUSSION TITLE
The PHP5 Password Hash Generator is not working

### DISCUSSION THREAD

**Discussion by iplocker** [prasathmani/tinyfilemanager#712] [General]
Hello.

The https://tinyfilemanager.github.io/docs/pwd.html is not working .

Thanks

**Comment by prasathmani** [comment]
verified, I able to access the link and generate the password hash. let me know still facing issue.

**Reply by iplocker** [reply]
Fixed indeed , just if we want to be perfect we have to update and the beside link : https://prnt.sc/26mj75q .
Thanks

**Reply by prasathmani** [reply]
thanks, will update.
